docs/source/related_topics/technical/hosting-overview.md

```diff
@@ -0,0 +1,208 @@
+```python
```

Comment:
a lot of this was chopped out of other files. the point is that people who are 'getting started' don't need to know how this stuff in this level of detail, but people who are serverops for a production server do.